plugins/woocommerce-blocks/assets/js/blocks/order-confirmation/additional-fields/edit.tsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import { useBlockProps } from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
+import {
+	ADDITIONAL_FORM_FIELDS,
+	CONTACT_FORM_FIELDS,
+} from '@woocommerce/block-settings';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+const Edit = (): JSX.Element => {
+	const blockProps = useBlockProps( {
+		className: 'wc-block-order-confirmation-additional-fields',
+	} );
+
+	const additionalFields = {
+		...ADDITIONAL_FORM_FIELDS,
+		...CONTACT_FORM_FIELDS,
+	};
+
+	return (
```

Comment:
We should do the same for the addresses, not sure if this can be re-used?

Reply:
Can be a follow up.

Reply:
Done in https://github.com/woocommerce/woocommerce/pull/43449/commits/1877ec4f4de101bfbc4df5d86ccd8d9b20b5ac63